packages/store/src/internal/internals.ts

```diff
@@ -72,6 +74,10 @@ export interface StatesAndDefaults {
   states: MappedStore[];
 }
 
+export interface StateInstanceInternal<T = any> {
+  [methodName: string]: (ctx: StateContext<T>, action: ActionType) => any;
```

Comment:
These methods are supposed to be private. They should only be called by the framework.